src/Microsoft.Health.Dicom.Core/Exceptions/DicomElementValidationException.cs

```diff
@@ -5,30 +5,39 @@
 
 using Dicom;
 using EnsureThat;
+using Microsoft.Health.Dicom.Core.Features.Validation;
 
 namespace Microsoft.Health.Dicom.Core.Exceptions
 {
     public class DicomElementValidationException : ValidationException
     {
-        public DicomElementValidationException(string name, string value, DicomVR vr, string message)
+        public DicomElementValidationException(ValidationErrorCode errorCode, string name, DicomVR vr, string message)
+           : this(errorCode, name, vr, message, null)
+        {
+        }
+
+        public DicomElementValidationException(ValidationErrorCode errorCode, string name, DicomVR vr, string message, string value)
```

Comment:
Is it fixed? From the caller, there is till this constructor that takes some params that can be null.

Why don't you make each constructor set message explictly instead

Reply:
sure, solved.